ui/components/component-library/avatar-icon/__snapshots__/avatar-icon.test.js.snap

```diff
@@ -0,0 +1,15 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`AvatarIcon should render correctly 1`] = `
+<div>
+  <div
+    class="box mm-avatar-base mm-avatar-base--size-md mm-avatar-icon box--display-flex box--flex-direction-row box--justify-content-center box--align-items-center box--color-primary-default box--background-color-primary-muted box--border-color-transparent box--border-style-solid box--border-width-1"
```

Comment:
I think that breaks our class name convention of `mm-component-name`?

Reply:
Oh is it? I think we should keep to the convention of `mm-component-name` even for base components imo. I feel like that it is a clear pattern that is adhered to across the extension codebase. Exceptions may create more confusion than benefit? I also feel it can help with discovery from a browser and creates a pattern in engineers minds? I thought our conversation was about constants but my memory might be failing me.

Reply:
Cool. Could create a ticket and just pop it under the house keeping epic. Doesn't have to be in depth but will help with our sprint points allocation and tracking efforts